cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,141 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 6.2.1 <http://docs.cask.co/cdap/6.2.0/index.html>`__
+=============================================================
+
+Summary
+-------
+
+This release introduces a number of new features, improvements, and bug fixes to CDAP. Some of the main highlights of the release are:
+
+1. **Joiner Performance Improvements**
+    - Implemented performance improvements to joiner plugins. Joins can now also be performed in-memory if one side is small, and behavior on null keys can be chosen by the user.
+
+2. **Aggregator Plugin Performance Improvements**
+    - Improved aggregator performance for Spark engine.
+
+
+New Features
+------------
+
+- :cask-issue:`CDAP-16708` - Added a new AutoJoiner API for plugins to implement. The new API leaves implementation details up to the application, which can perform join optimizations that were not possible with the older Joiner API.
+
+- :cask-issue:`CDAP-16530` - Fixed joiner output schema generation to be deterministic, using the same ordering as they had in the input data.
+
+- :cask-issue:`CDAP-16711` - Added the ability for Joiner plugins to specify whether null keys should match other null keys.
+
+- :cask-issue:`CDAP-16875` - Added two new properties to the Joiner plugin -- 'Join on Null Values' and 'Inputs to Load in Memory'. The plugin also uses new join capabilities that make it more resilient to running out of memory.
+
+- :cask-issue:`CDAP-16461` - Added Spark parameter to limit Spark block size to prevent issues with joins
+
+- :cask-issue:`CDAP-16855` - Introduced a new aggregator API to achieve better performance for spark engine.
+
+
+Improvements
+------------
+
+- :cask-issue:`CDAP-16455` - Include logs emitted from the job main class as the Dataproc job logs
+
+- :cask-issue:`CDAP-16682` - When backend is slow to respond to requests from UI, we now show a snackbar saying there's a delay.
+
+- :cask-issue:`CDAP-16690` - Added revamped preview tab with new Record view for large schemas.
+
+- :cask-issue:`CDAP-16835` - Added support for upgrading applications via REST API. Example usage is to upgrade all pipelines in a namespace to use latest available artifacts.
+
+- :cask-issue:`CDAP-16836` - -l option is introduced in CDAP CLI to take in URI in the new format "http[s]://hostname:[port]
```

Comment:
Introduced a ``-l`` option in the CLI...

Reply:
fixed.